drivers/SmartThings/zigbee-window-treatment/src/init.lua

```diff
@@ -15,6 +15,23 @@
 local capabilities = require "st.capabilities"
 local ZigbeeDriver = require "st.zigbee"
 local defaults = require "st.zigbee.defaults"
+local window_preset_defaults = require "st.zigbee.defaults.windowShadePreset_defaults"
+
+local function init_handler(self, device)
```

Comment:
Like I mentioned in the [CHAD-15784](https://smartthings.atlassian.net/browse/CHAD-15784?focusedCommentId=1637968), we need this.
device:emit_event(capabilities.windowShadePreset.supportedCommands({"presetPosition", "setPresetPosition"}, { visibility = { displayed = false }})) in init handler.